Assets/File.php

```diff
@@ -472,36 +472,36 @@ public function canDelete($member = null) {
 	public function getCMSFields() {
 		$path = '/' . dirname($this->getFilename());
 
-		$fields = FieldList::create([
-			HeaderField::create('TitleHeader', $this->Title, 1),
-			LiteralField::create("ImageFull", $this->PreviewThumbnail()),
-			TextField::create("Name", $this->fieldLabel('Filename')),
-			ReadonlyField::create(
-				"Path",
-				_t('AssetTableField.PATH', 'Path'),
-				(($path !== '/.') ? $path : '') . '/'
-			)
-		]);
-
-		if ($this->getIsImage()) {
-			$fields->push(ReadonlyField::create(
-				"DisplaySize",
-				_t('AssetTableField.SIZE', "File size"),
-				sprintf('%spx, %s', $this->getDimensions(), $this->getSize())
-			));
-			$fields->push(HTMLReadonlyField::create(
-						'ClickableURL',
-						_t('AssetTableField.URL','URL'),
-						sprintf('<a href="%s" target="_blank">%s</a>', $this->Link(), $this->Link())
-			));
-		}
-		$fields->push(HiddenField::create('ID', $this->ID));
+		$width = (int)Config::inst()->get('Image', 'asset_preview_width');
```

Comment:
You're right, this only gets the icon of the File/Folder, I hadn't considered removing the Image Scaling calls.